constellations/gemini-user/src/internship/internship.sass

```diff
@@ -20,26 +25,47 @@
         & img
             width: 120px
             margin-right: 60px
-        
+    
         & p
             margin: 0
-     
 
-    #past-intern
-        border-collapse: collapse
-        border-spacing: 2px
-        width: 80%
-        margin: 30px auto
+    .intern-main
+        display: flex
+        flex-direction: row
+        & p
+            margin: 20px 20px 20px 0
+        .name
+            width: 120px
+        .school
+            width: 250px
+
+    #last-paragraph
+        margin-bottom: 300px
 
+    @media only screen and (max-width: 700px)
+        & h1, h4, p
+            text-align: center
 
-        & caption
-            color: $colorTurquoise
+        .container-main
+            display: inline
+            flex-direction: column
+            align-items: center
 
-        & td, th
-            border: 1px solid
-            border-color: #dddddd
-            padding: 8px
-            text-align: left       
+            & img
+                margin-left: auto
+                margin-right: auto
+                margin-bottom: 0
+                width: 90%
```

Comment:
The images are bit too big in mobile view.
I think a width of 180px is good enough.
If you want to center using margin auto, add the following properties:
```
display: block
margin: 0 auto
width: _____
```